framework/Metrics/__init__.py

```diff
@@ -27,6 +25,7 @@
 from .PairwiseMetric import PairwiseMetric
 from .CDFAreaDifference import CDFAreaDifference
 from .PDFCommonArea import PDFCommonArea
+#from .RACDistance import RACDistance
```

Comment:
I would like to keep this line commented out because, I woul like to add the new metric once this PR is merged